src/main/scala/org/broadinstitute/dsde/rawls/dataaccess/GraphSubmissionDAO.scala

```diff
@@ -69,11 +70,15 @@ class GraphSubmissionDAO extends SubmissionDAO with GraphDAO {
       val workspace = getWorkspaceVertex(db, workspaceNamespace, workspaceName).getOrElse(throw new IllegalArgumentException(s"workspace ${workspaceNamespace}/${workspaceName} does not exist"))
       val submissionVertex = setVertexProperties(submission, addVertex(db, null))
       workspace.addEdge(ExecutionEdgeTypes.submissionEdgeType, submissionVertex)
-      submission.workflow.foreach { workflow =>
+      submission.workflows.foreach { workflow =>
```

Comment:
So this save creates new submission vertexes and edges each time right?  I guess the submission never changes, but we may want to throw an exception if you try to save more than once